src/ts/contentScripts/dispatcher/index.ts

```diff
@@ -20,6 +20,6 @@ const dispatchMap = new Map([
 
 browser.runtime.onMessage.addListener(command => dispatchMap.get(command)?.())
 
-document.addEventListener('keyup', ev => {
-    if (ev.key === guard) replaceFuzzyDate()
+document.addEventListener('keypress', ev => {
+    if (ev.key === guard) setTimeout(function(){ replaceFuzzyDate() },0)
```

Comment:
nit: I don't believe you need to wrap `replaceFuzzyDate` into additional function, I think you can just do:
`setTimeout(replaceFuzzyDate,0)`

Reply:
ah, another nit is that there should be space after coma :p

Reply:
you are right, fixed it 😉